tests/xharness/Jenkins.cs

```diff
@@ -209,6 +210,8 @@ IEnumerable<TestData> GetTestData (RunTestTask test)
 				yield return new TestData { Variation = "AssemblyBuildTarget: SDK framework (debug, profiling)", MTouchExtraArgs = "--assembly-build-target=@sdk=framework=Xamarin.Sdk --assembly-build-target=@all=staticobject", Debug = true, Profiling = true, MonoNativeLinkMode = MonoNativeLinkMode.Static, MonoNativeFlavor = flavor };
 
 				yield return new TestData { Variation = "Release", MTouchExtraArgs = "", Debug = false, Profiling = false, MonoNativeLinkMode = MonoNativeLinkMode.Static };
+				yield return new TestData { Variation = "Release: SGenConc", MTouchExtraArgs = "", Debug = false, Profiling = false, MonoNativeLinkMode = MonoNativeLinkMode.Static, EnableSGenConc = true};
+				yield return new TestData { Variation = "Debug: SGenConc", MTouchExtraArgs = "", Debug = true, Profiling = false, MonoNativeLinkMode = MonoNativeLinkMode.Static, EnableSGenConc = true};
```

Comment:
I don't think there's much (if any) interaction between Debug and Release vs sgen in concurrent mode.

Beside the non-trivial extra time to add both, it makes the test run more susceptible to random errors - leading to spending more (manual) time reviewing and green'ifying builds.

Reply:
@spouliot I'm trying to match the matrix tests in QA, if we are ok of not adding it because we do not need it, I'm more than happy to drop it and we document the reason :)

@GouriKumari any feedback on this?

Reply:
@mandel-macaque it's not about adding it (or not) to the matrix - it's adding it twice when there's nothing suggesting it would give use more than 1x more coverage. That does not scale - no pros (e.g. extra issues found) & lots of cons (e.g. manual work, delay in results)

Reply:
In my opinion, enabling sgen concurrent with Debug will suffice. I haven't found any additional issues or differences in results recently by adding this configuration. These configs were added when the sgen concurrent were not public to users.

Reply:
@spouliot @GouriKumari ok, so we have an agreement, I'll remove the extra one